scripts/updateReactNativeVersion.js

```diff
@@ -64,9 +64,40 @@ async function main(version) {
     },
   });
 
-  const updated = JSON.stringify(pkgJson, null, 2);
-  console.log(`Writing update package.json to ${PKG_JSON_PATH}:\n\n${updated}`);
+  let updated = JSON.stringify(pkgJson, null, 2);
+  console.log(
+    `\nWriting update template/package.json to ${PKG_JSON_PATH}:\n\n${updated}`,
+  );
   fs.writeFileSync(PKG_JSON_PATH, updated);
+
+  // 2. Update the scripts.version field in the top level package.json. This lets us leverage
+  //    the https://registry.npmjs.org/@react-native-community/template API:
+  //
+  //  "name": "@react-native-community/template",
+  //  "dist-tags": {
+  //    ...
+  //    "0.75-stable": "0.75.0-rc.0"
+  //  },
+  //  "versions": {
+  //    "0.75.0-rc.0": {
+  //      "name": "@react-native-community/template",
+  //      "version": "0.75.0-rc.0",
+  //      "scripts": {
+  //        "version": "0.75.1"
```

Comment:
it's not clear to me how this will work. 😅 but surely it is because I don't know `yarn` very well.

AFAIK, the `scripts` property in the package.json is used to define some commands we can run with `yarn <script>`.

What will `yarn version` do in this case? 🤔